digitalfabrik/integreat-app
# 2173: Send language to iOS VoiceOver

### Short description


As part of the test of our accessibility from Pfennigparade a while ago, we got the feedback that the app doesn't send the language chosen in the app to OS so that you might get e.g. German texts read with an English pronunciation.

Android does a good job of that natively already, and the prop [accessibilityLanguage](https://reactnative.dev/docs/accessibility#accessibilitylanguage-ios) can be used to achieve the same thing in iOS.

### Proposed changes



- Added the prop accessibilityLanguage in many places.
- Wrote a little function telling us what language the text in the header should be.

### Side effects


- I turned the Pressable in the NewsItemList into two Pressables that lead to the same place because the first half is in the language that the app is in and the second part ("Continue") is in the system language. I didn't see a visual or behavioural difference but maybe I missed something.

### Testing


1. Be on an iPhone (VoiceOver doesn't exist on emulators, yay.)
2. Turn on VoiceOver
3. Set the app language to one that is different from your phone's language
4. Walk around through the app

Alternatively, you can edit the `HeaderBox` to show the title and the language, and that should be good enough for a test.

### Resolved issues



Fixes: #2173
